### ISSUE TITLE
What ports need to be open

### ISSUE THREAD

**Issue by ryanramage** [mafintosh/hypercore-archiver#30]
This is great. I am trying to run this behind a firewall and I am having a hard time finding what protocol(tcp or udp) and ports need to be open for this to be able to work so peers can connect. Thanks!

**Comment by joehand** [comment]
The default port is 3282 on both tcp and udp. You can pass another port to the swarm if you want to specify:

```js
var swarm = require('hypercore-archiver/swarm')
swarm(archiver, {port: 60234})
```

**Comment by joehand** [comment]
Oops, nevermind. Its hardcoded right now: https://github.com/mafintosh/hypercore-archiver/blob/master/swarm.js#L29

**Comment by ryanramage** [comment]
ok, thanks! I added a quick commit to make it behave as you described.

**Comment by joehand** [comment]
Released in v4.4.0
